src/test/java/edu/harvard/iq/dataverse/export/ddi/exportfull.xml

```diff
@@ -144,7 +144,6 @@
       <notes type="NotesType" subject="NotesSubject">NotesText</notes>
     </method>
     <dataAccs>
-      <notes type="DVN:TOU" level="dv">CC0 Waiver</notes>
```

Comment:
I'm not sure what's going on here. For DDI, we no longer export the Terms of Use? This seems like a potential backward incompatibility.

Reply:
Not sure if removing it is best, but this is a Dataverse specific note that isn't used by others and we'd at least have to update what the content is if we keep it (Is it the license URL or name or ? for standard licenses? Is it 'Custom Data Terms' for custom licenses or just the Terms of Use field itself in that case (nominally the 'Custom Dataset Terms' includes the other fields as well which we haven't ever been including.)

With the URLs, we could do what ICPSR does though I understand that is also not standard and I'm not sure DDI has a standard answer yet.